### PR TITLE
Geo IP Docs

### PR DESCRIPTION
**Pull request by rbibby**

### CODE REVIEW

File: source/network/load-balancers/components/geo-ip.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Geo IP
+
+You can use the Geo IP settings on listeners to block countries or continents from accessing your load balancer (deny list) or only allow certain countries or continents to access your load balancer (allow list).
```
Comment (bluemorbo) [inline source/network/load-balancers/components/geo-ip.md]: You can use the Geo IP settings on a listener to allow or deny requests from specific countries and continents.

**Review by miff2000 — APPROVED** [review]

**Comment by miff2000** [comment]
Getting Dane to check why the CI failed

**Comment by bluemorbo** [comment]
@miff2000 Is there any update on the CI failures?

**Comment by Xiol** [comment]
> @miff2000 Is there any update on the CI failures?

Not yet - I'm having a bit of a nightmare with it. I think it's because the changes are coming from a fork and not a branch of this repository, but nothing I do is working to give the Github token write access to the organisation packages (probably for very good security reasons).

File: source/network/load-balancers/components/geo-ip.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## List Type
+
+You can toggle the list between an allow and deny using the "List Type" toggle.
+
+* Allow List - Only users in the countries/continents specified will be allowed to access your load balancer
+* Deny List - Any user in a country/continent specified will be blocked from being able to access your load balancer
+
+## EU Filtering
+
+As well as being able to specify specific countries or continents, you can also add all countries within the EU to the list as a group. For instance, if you only want EU customers to be able to access your website you could have an allow list setup with EU filtering included.
```
Comment (Xiol) [inline source/network/load-balancers/components/geo-ip.md]: Could be worth adding another sentence about the UK here just for clarity, e.g. "Note that the EU list does not include the United Kingdom".

**Comment by Xiol** [comment]
UAT environment has been fixed: https://pr-923.docsuat.devops.ukfast.co.uk/network/load-balancers/components/geo-ip.html

**Review by Xiol — APPROVED** [review]

**Comment by rbibby** [comment]
@miff2000 is there anything left for me to do here or can this be merged?

There's still 1 out of 3 checks failing which I'm not sure about

**Comment by Xiol** [comment]
Can ignore the failing CI check, it's just a holdover from all the CI breakage I was doing. Not sure how to bin it off.

Will merge as it's been approved previously.

**Pull request merged**